utils/crypto.utils.js

```diff
@@ -0,0 +1,36 @@
+const crypto = require('crypto')
+
+const config = {
+  initialSalt: 'd75535de98ecea315854491c8d036f8f',
+  iterations: 100000,
+  hashBytes: 32,
+  digest: 'sha512',
+}
+
+const hashArgs = [config.iterations, config.hashBytes, config.digest]
+
+const hashString = (stringToHash, useInitialSalt) => {
```

Comment:
this one is good: `hashString('string')`
but this one is confusing: `hashString('string', true)`

could we use an options object for the last variable here? that way the intent is a bit clearer.

`const hashString = (stringToHash, { useInitialSalt } = {}) => {`

then to call it it looks like:

`hashString('string', {useInitialSalt: true})`